### PR TITLE
chore(deps): Bump @nextcloud packages

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ <skjnldsv@protonmail.com>

### CODE REVIEW

File: lib/components/UploadPicker.vue [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
 <script>
 import { getNewFileMenuEntries } from '@nextcloud/files'
-import { getUploader } from '../index.ts'
+import { getUploader } from '../index.js'
```
Comment (skjnldsv) [inline lib/components/UploadPicker.vue:76]: js? There is no js file? :thinking:

**Comment by skjnldsv** [comment]
Any news @susnux ? :see_no_evil:

**Comment by susnux** [comment]
Yes with auth 2.1.0 it works, but I get one jest error (network issue, somehow the axios mock does not work)

**Comment by skjnldsv** [comment]
> Is changelog expected to work?

Not for this, the failing step should be skipped in the workflow file

File: lib/components/UploadPicker.vue (hunk 1 above)
Comment (skjnldsv) [reply lib/components/UploadPicker.vue:76]: This feel so weird

Comment (susnux) [reply lib/components/UploadPicker.vue:76]: Yes... :disappointed:

**Comment by susnux** [comment]
All green^^

**Comment by skjnldsv** [comment]
:rocket: :rocket:

**Comment by susnux** [comment]
But I would like to test this with an app first (I think currently photos is the only one using it?).
To make sure it does work as expected :)

**Comment by skjnldsv** [comment]
> But I would like to test this with an app first (I think currently photos is the only one using it?). To make sure it does work as expected :)

The cypress tests are also here for it
https://cloud.cypress.io/projects/v24ts6/runs/483/test-results

Otherwise yep, Photos

**Comment by skjnldsv** [comment]
We can merge and release a new beta

**Comment by susnux** [comment]
@skjnldsv I have now tested that photos can be at least compiled :heavy_check_mark: 
(The tests do not cover that importing the module is possible, so I wanted tot test this ;-) ).

**Review by susnux — APPROVED** [review]

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/nextcloud-upload/releases/tag/v1.0.0-beta.9

**Pull request merged**